chia/wallet/wallet_state_manager.py

```diff
@@ -360,11 +367,13 @@ async def get_unused_derivation_record(
             # If we have no unused public keys, we will create new ones
             unused: Optional[uint32] = await self.puzzle_store.get_unused_derivation_path()
             if unused is None:
+                self.log.debug("No unused paths, generate more ")
                 await self.create_more_puzzle_hashes()
+                # Now we must have unused public keys
+                unused = await self.puzzle_store.get_unused_derivation_path()
+                assert unused is not None
 
-            # Now we must have unused public keys
-            unused = await self.puzzle_store.get_unused_derivation_path()
-            assert unused is not None
+            self.log.debug("Fetching derivation record for: %s %s %s", unused, wallet_id, hardened)
```

Comment:
nit: we don't seem to be entirely consistent with format strings in the codebase, but generally f-strings are preferred.

Reply:
f-strings are good, but the problem with using f-string formatting in logging is those variables get evaluated even if logging level is above. See [this](https://github.com/Chia-Network/chia-blockchain/pull/11184).